app/frontend/pages/home.js

```diff
@@ -6,6 +6,14 @@ import { FormattedMessage, injectIntl } from 'react-intl'
 
 import { forcePublic } from '~base/middlewares/'
 
+import phone from '../../public/img/phone.png'
```

Comment:
Change this imports to


import Image from "~base/components/image"

<Image src={"/public/img/racam.png"} alt="ra cam" />